src/pages/quizzes.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { useState } from "react"
+import { Box, Flex, Heading, Icon, Stack, Text } from "@chakra-ui/react"
+import { graphql, PageProps } from "gatsby"
+import { useTranslation } from "gatsby-plugin-react-i18next"
+import { FaGithub } from "react-icons/fa"
+
+import ButtonLink from "../components/ButtonLink"
+import PageHero from "../components/PageHero"
+import PageMetadata from "../components/PageMetadata"
+import Translation from "../components/Translation"
+import { Content } from "../components/SharedStyledComponents"
+import FeedbackCard from "../components/FeedbackCard"
+import QuizWidget from "../components/Quiz/QuizWidget"
+import QuizzesList from "../components/Quiz/QuizzesList"
+import QuizzesModal from "../components/Quiz/QuizzesModal"
+import QuizzesStats from "../components/Quiz/QuizzesStats"
+import { QuizzesHubContext } from "../components/Quiz/context"
+
+import { useLocalStorage } from "../hooks/useLocalStorage"
+
+import { getImage } from "../utils/image"
+import { trackCustomEvent } from "../utils/matomo"
+
+import { INITIAL_QUIZ, USER_STATS_KEY } from "../constants"
+
+import { CompletedQuizzes, QuizStatus } from "../types"
+
+import allQuizzesData, {
+  ethereumBasicsQuizzes,
+  usingEthereumQuizzes,
+} from "../data/quizzes"
+
+// Contains each quiz id as key and <boolean, number> to indicate if its completed and the highest score in that quiz
+// Initialize all quizzes as not completed
+const INITIAL_COMPLETED_QUIZZES: CompletedQuizzes = Object.keys(
+  allQuizzesData
+).reduce((object, key) => ({ ...object, [key]: [false, 0] }), {})
+
+const INITIAL_USER_STATS = {
+  score: 0,
+  average: [],
+  completed: JSON.stringify(INITIAL_COMPLETED_QUIZZES),
```

Comment:
I'm saying this because we are passing the entire `INITIAL_USER_STATS` object to `useLocalStorage`. And that hook is going to stringify the whole object for you.

`stringify` will stringify the nested objects correctly:
```tsx
const INITIAL_USER_STATS = {
  score: 0,
  average: [],
  completed: INITIAL_COMPLETED_QUIZZES,
}

JSON.stringify(INITIAL_USER_STATS)
// output: '{"score":0,"average":[],"completed":{"what-is-ethereum":[false,0],"what-is-ether":[false,0],"web3":[false,0],"wallets":[false,0],"security":[false,0],"nfts":[false,0],"layer-2":[false,0],"merge":[false,0]}}'
```